vllm/worker/model_runner.py

```diff
@@ -74,6 +75,22 @@ def __init__(
         # cache in_wsl result
         self.in_wsl = in_wsl()
         self.kv_cache_dtype = kv_cache_dtype
+        self.kv_quant_params = self.load_kv_quant_params(
+            model_config,
+            kv_quant_params_path) if model_config is not None else None
+
+    def load_kv_quant_params(self, model_config: ModelConfig,
+                             kv_quant_params_path: str) -> List[List[float]]:
+        num_layers = model_config.hf_config.num_hidden_layers
+        kv_quant_params = []
+        for i in range(num_layers):
+            # default quant scales and zero points for kv int8 quant
+            kv_quant_param = [1.0, 0.0, 1.0, 0.0]
+            if kv_quant_params_path is not None:
+                path = kv_quant_params_path + f"/layers.{i}.past_kv_scale.0.weight"
```

Comment:
Does the path name relate to model?

Reply:
No, it is just a path that save the exported kv quant params.